sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousExecution.scala

```diff
@@ -172,9 +172,9 @@ class ContinuousExecution(
     }
 
     withNewSources.transformAllExpressionsWithPruning(_.containsPattern(CURRENT_LIKE)) {
-      case (_: CurrentTimestamp | _: CurrentDate) =>
-        throw new IllegalStateException(
-          "CurrentTimestamp and CurrentDate not yet supported for continuous processing")
+      case (_: CurrentTimestamp | _: CurrentDate | _: LocalTimestamp) =>
```

Comment:
How about `Now`? Should we just check `CurrentTimestampLike`?

Reply:
`CurrentDate` and `LocalTimestamp` not extend `CurrentTimestampLike` now.

Reply:
hmm, I haven't realized that you threw away inheritance from CurrentTimestampLike in https://github.com/apache/spark/pull/33258/commits/945a1d66c3a2d3292a894da71e5513a7bf5e3294

How about this one https://github.com/apache/spark/pull/33258/files#diff-7c879c08d2f379c139d5229a88857229ae69bb48f0a138a3d64e1b2dde3502feR420 . It doesn't check LocalTimestamp now, correct?

Reply:
> We should replace `CurrentTimestamp` with `CurrentTimestampLike`, though this is not related to this PR.

OK